std/array.d

```diff
@@ -380,6 +380,95 @@ if (isNarrowString!String)
     )));
 }
 
+/**
+ * Returns a static array from the stack with length of s and initializes
+ * it with copies of the elements of range $(D r). The remainder of the
+ * array is default-initialized.
+ *
+ * Params:
+ *      r = The range (or aggregate with $(D opApply) function) whose elements
+ *      are copied into the stack-allocated array. Must be as long or shorter
+ *      than s.
+ * Returns:
+ *      An initialized static array
+ */
+ForeachType!Range[s] array(size_t s, Range)(Range r)
+if (isInputRange!Range && !isInfinite!Range)
```

Comment:
You don't care whether the range is finite - you take a finite subset anyway.

Reply:
That's if I change it to check that. Now it goes unchecked.